pkg/generator/util_test.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2018 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package generator
+
+import (
+	"testing"
+)
+
+func TestToPlural(t *testing.T) {
+	var pluralTestData = []struct {
+		input    string
+		expValue string
+	}{
+		{"value", "values"},
+		{"values", "valueses"},
```

Comment:
Just a minor nit but can you change all the words so that their plurals are proper words. It just makes the test cases clearer:
`class, classes`, `box, boxes`, `brush, brushes`, `watch, watches`